### PR TITLE
fix: Align motion match

### LINKED ISSUE #26519: Prop `className` did not match.
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[Edit on CodeSandbox](https://codesandbox.io/s/jovial-glitter-57vt1?file=/src/App.tsx)

### Steps to reproduce
- 1.复制网址
- 2.打开控制台

### What is expected?
无错误

### What is actually happening?
Warning: Prop `className` did not match. Server: "ant-form-item-explain" Client: "ant-form-item-explain show-help-appear show-help"

| Environment | Info |
|---|---|
| antd | 4.6.2 |
| React | 16.13.1 |
| System | macos |
| Browser | chrome |

---
使用ssr触发className不一致

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

resolve #26519

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Fix Motion related issue like Upload align flash and Form.Item with `help` ssr issue.       |
| 🇨🇳 Chinese |    修复 Motion 相关问题例如 Upload 对齐闪烁与 Form.Item 配置 `help` 的 ssr 问题。      |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by zombieJ** [comment]
先过一下 Tooltip，如果没问题 antd 下个版本再批量把 `rc-trigger` 升级上来。

File: package.json [hunk 1]
```diff
@@ -143,10 +143,10 @@
     "rc-table": "~7.9.2",
     "rc-tabs": "~11.6.0",
     "rc-textarea": "~0.3.0",
-    "rc-tooltip": "~4.2.0",
+    "rc-tooltip": "~5.0.0",
     "rc-tree": "~3.9.0",
     "rc-tree-select": "~4.1.1",
-    "rc-trigger": "~4.4.0",
+    "rc-trigger": "^5.0.2",
```
Comment (afc163) [inline package.json:149]:
```suggestion
    "rc-trigger": "~5.0.2",
```

Comment (afc163) [reply package.json]: 之前是 ~ ，就保持吧，这个升级的次数不会很多。

**Comment by zombieJ** [comment]
@afc163 pls help to check~

**Review by afc163 — APPROVED** [review]

**Comment by afc163** [comment]
体积超标了。

**Comment by afc163** [comment]
test ci 也挂了。

**Comment by zombieJ** [comment]
> 先过一下 Tooltip，如果没问题 antd 下个版本再批量把 `rc-trigger` 升级上来。

不会太久的，这周发完后如果没有人报我下个 patch 统一升上来。免得一下子挂太多了。

**Comment by zombieJ** [comment]
test 是 rc-image 的 async，看起来不是稳定出现。我看看能不能调整一下：
* test_dom: https://app.circleci.com/pipelines/github/ant-design/ant-design/17028/workflows/24dd48d7-ef33-4d3b-8d23-51eb53889f28/jobs/230700
* test_es: https://app.circleci.com/pipelines/github/ant-design/ant-design/17025/workflows/980059a2-564a-4686-9d52-bd02a007b4cd/jobs/230664

**Pull request merged**